### PR TITLE
Added Clown and Mime Offices

### PR DESCRIPTION
**Pull request by Spacedong**
Adds Clown and Mime offices.

Also adds recharger in Escape Security Checkpoint.

http://vgy.me/b8FZnB.png

http://vgy.me/n21Iw6.png

http://vgy.me/FhJ2sa.png

### CODE REVIEW

**Comment by Spacedong** [comment]
alright now how do I add a changelog to this sucker

File: config/initial_config/admins.txt [hunk 1]
```diff
@@ -8,3 +8,5 @@
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
 Admin = Rank
+
```
Comment (AliiTea) [inline config/initial_config/admins.txt]: :^)

**Comment by blobbernaut** [comment]
as server owner, creator of spacestation 13, and president of the united states, I vote yes

**Comment by Carbonhell** [comment]
PR is ok code and map wise

**Comment by Nebyab** [comment]
then itsallgood

**Comment by Nokington** [comment]
As long as someone checked to make sure the map isn't fucked up like last time. Then I will :+1:

**Comment by TDcoolguy300** [comment]
Carbon has checked it out mapwise

**Comment by TDcoolguy300** [comment]
Headmin approval, checked it out codewise and is good

**Comment by Kokojos** [comment]
No mime statues?

**Pull request merged**